doc/scripts/extract_content.py

```diff
@@ -37,14 +38,19 @@ def copy_if_different(src, dst):
         return
     shutil.copyfile(src, dst)
 
-def get_rst_files(dest, dir):
+def get_files(all, dest, dir):
     matches = []
     for root, dirnames, filenames in os.walk('%s/%s' %(ZEPHYR_BASE, dir)):
         if ZEPHYR_BUILD:
             if os.path.normpath(root).startswith(os.path.normpath(ZEPHYR_BUILD)):
                 # Build folder, skip it
                 continue
-        for filename in fnmatch.filter(filenames, '*.rst'):
+        if all:
+            wildcard = '*'
+        else:
+            wildcard = '*.rst'
+
+        for filename in fnmatch.filter(filenames, wildcard):
```

Comment:
Super subjective, but in case you haven't tried out the Python ternary yet, you can make this

    for filename in fnmatch.filter(filenames, '*' if all else '*.rst'):

Reply:
thanks for the tip! I will do that

Reply:
@ulfalizer done, and tested, thank you!